### DISCUSSION TITLE
How do I set point coordinates of a line object in YAML?

### DISCUSSION THREAD

**Discussion by McBenac** [HASwitchPlate/openHASP#709] [Q&A]
I'm using a display which I update in Home Assistant.

I have a line object:
`{"id":5,"obj":"line","points":[[40,227],[45,227]]}
`

and I want to adjust the points in the YAML file:
```
    - obj: "p3b5"
      properties:
       "points": '[[40,247],[145,247]]'
```

But nothing happens with the line.

I also tried this:
```
    - obj: "p3b5"
      properties:
        "points": [[40,247],[145,247]]
```

but then Home Assistant protests saying:

> Configuration warnings
> Invalid config for 'openhasp' at openhasp.yaml, line 56: template value should be a string for dictionary value 'openhasp->plate->objects->65->properties->points', got [[40, 247], [145, 247]]

Please help me understand how to set this array of coordinates for the points property in the YAML file.

**Comment by fvanroie** [comment]
Try using the `jsonl`  [pseudo property](https://www.openhasp.com/0.7.0/design/objects/#common-properties), something like:
```yaml
    - obj: "p3b5"
      properties:
         "jsonl": '{"points":[[40,227],[45,227]]}'

**Comment by McBenac** [comment]
I'm so sorry. My fault! I had set the _wrong page_ (p3 was wrong). 🫣

This was the correct way:
```
    - obj: "p6b5"
      properties:
        "points": [[40,{{ y }}],[45,{{ y }}]]
```
